account_analytic_parent/models/account_analytic_account.py

```diff
@@ -11,15 +11,26 @@
 
 class AccountAnalyticAccount(models.Model):
     _inherit = "account.analytic.account"
+    _parent_name = "parent_id"
+    _parent_store = True
+    _rec_name = 'display_name'
+    _order = 'display_name'
```

Comment:
It should include `parent_path`

Reply:
Yep, lost one commit, sorry

Reply:
@pedrobaeza actually I'm not sure about `display_name` vs how it's implemented elsewhere using `complete_name`

Reply:
Well, the problem is that complete_name can be translation aware, so `parent_path` assures the proper order. Look for other implementations of this in Odoo core.

Reply:
I see... `complete_name` is a computed stored field, isn't it? Then translatable stuff doesn't work there.

Reply:
And it can be used for ordering.

Reply:
Yep, computed + stored. So, I'm thinking of changing this to https://github.com/odoo/odoo/blob/bdfef60d60d78c5e7360fdabe80e70ea208e6078/addons/analytic/models/analytic_account.py#L50 - what do you thing?

Reply:
OK, use `_order = 'complete_name'`